pkg/cluster/config/types.go

```diff
@@ -96,4 +96,7 @@ type Networking struct {
 	//
 	// Defaults to 127.0.0.1
 	APIServerAddress string
+	// PodSubnet is the CIDR used for pod IPs
+	// kind will select a default if unspecified
+	PodSubnet string
```

Comment:
TODO: we should validate this by checking `net.ParseCIDR`

Reply:
I think that kubedm does it
https://github.com/kubernetes/kubernetes/blob/59cedcf5415c1419d03a443d2a27600e1d3ab8fc/cmd/kubeadm/app/apis/kubeadm/validation/validation.go#L390

Reply:
yeah they're doing a bit more. we're validating that it's at least a subnet for now :-)

Reply:
TODO is done